auto_nag/scripts/mismatch-priority-tracking-beta.py

```diff
@@ -0,0 +1,69 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.bugzilla.utils import getVersions
+
+
+class MismatchPrioTrackBeta(BzCleaner):
+    def __init__(self):
+        super(MismatchPrioTrackBeta, self).__init__()
+
+    def description(self):
+        return 'Bug tracked for beta with a bad priority (P3, P4 or P5)'
+
+    def name(self):
+        return 'mismatch-priority-tracking'
+
+    def template(self):
+        return 'mismatch-priority-tracking.html'
+
+    def subject(self):
+        return self.description()
+
+    def ignore_date(self):
+        return True
+
+    def get_bz_params(self, date):
+        release_version, beta_version, central_version = getVersions()
+        value = ','.join(['', 'affected'])
+        params = {
+            'resolution': [
+                '---',
+                'FIXED',
+                'INVALID',
+                'WONTFIX',
+                'DUPLICATE',
+                'WORKSFORME',
+                'INCOMPLETE',
+                'SUPPORT',
+                'EXPIRED',
+                'MOVED',
+            ],
+            'priority': ['P3', 'P4', 'P5'],
+            'f1': 'OP',
+            'f2': 'cf_tracking_firefox' + beta_version,
+            'o2': 'anyexact',
+            'v2': ','.join(['+', 'blocking']),
+            'f3': 'cf_status_firefox' + beta_version,
+            'o3': 'anyexact',
+            'v3': value,
+            'o4': 'anyexact',
```

Comment:
o3 should be 'equals'

Reply:
it can be empty or affected, so, anyexact sounds good?

Reply:
I don't think so:
https://bugzilla.mozilla.org/buglist.cgi?f1=cf_status_firefox66&list_id=14492210&o1=anyexact&query_format=advanced&v1=%2Caffected&order=assigned_to%2Cproduct%2Cbug_id&limit=0

we've only 635 results here. I'd say that '---,affected' is the solution (we've more than 10,000 results with this string).

Reply:
OK, fixed in 09cdcb2